releasenotes/notes/rename-model-kwargs-to-invocation-kwargs-7b4ad0c4b05bd902.yaml

```
@@ -0,0 +1,32 @@
+---
+prelude: >
+    Replace this text with content to appear at the top of the section for this
+    release. This is equivalent to the "Highlights" section we used before.
+    The prelude might repeat some details that are also present in other notes
+    from the same release, that's ok. Not every release note requires a prelude,
+    use it only to describe major features or notable changes.
+upgrade:
+  - |
+    List upgrade notes here, or remove this section.
+    Upgrade notes should be rare: only list known/potential breaking changes,
+    or major changes that require user action before the upgrade.
+    Notes here must include steps that users can follow to 1. know if they're
+    affected and 2. handle the change gracefully on their end.
+features:
+  - |
+    List new features here, or remove this section.
+enhancements:
+  - |
+    change `model_kwargs` to `invocation_kwargs` in the PromptNode and PromptModel classes
+issues:
+  - |
+    List known issues here, or remove this section. For example, if some change is experimental or known to not work in some cases, it should be mentioned here.
+deprecations:
+  - |
+    List deprecations notes here, or remove this section. Deprecations should not be used for something that is removed in the release, use upgrade section instead. Deprecation should allow time for users to make necessary changes for the removal to happen in a future release.
+security:
+  - |
+    Add security notes here, or remove this section.
+fixes:
+  - |
+    Add normal bug fixes here, or remove this section.
```

```suggestion
enhancements:
  - |
    Change `model_kwargs` to `invocation_kwargs` in the PromptNode and PromptModel classes.
```

Hi @faaany , this is the last one with the release notes, sorry 🙂

---

no, this is my fault. Thanks for the correction!